test/e2e/specs/editor/various/block-comments.spec.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.use( {
+	blockCommentUtils: async ( { page, editor }, use ) => {
+		await use( new BlockCommentUtils( { page, editor } ) );
+	},
+} );
+
+test.describe( 'Block Comments', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.setGutenbergExperiments( [
+			'gutenberg-block-comment',
+		] );
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.deleteAllComments( 'block_comment' ),
+			requestUtils.setGutenbergExperiments( [] ),
+		] );
+	} );
+
+	test( 'can pin and unpin comments sidebar', async ( {
+		page,
+		blockCommentUtils,
+	} ) => {
+		const topBarButton = await blockCommentUtils.openBlockCommentSidebar();
```

Comment:
```suggestion
		const topBarButton = await blockCommentUtils.getBlockCommentTopBarButton();
```

It might be better to match the method name with what the method actually does.

Reply:
Method does two things, decided to go with `open` prefix, since that's the main use case for it.